docs/src/install-guide.md

```diff
@@ -1,6 +1,34 @@
 # Installation
 
-Kani must currently be built from source.
+To install the latest version of Kani, run:
+
+```bash
+cargo install --locked kani-verifier
```

Comment:
What happens if someone forgets to pass `--locked`

Reply:
_Probably_ nothing bad, it just uses newer versions of (currently 2) dependencies, instead of what's in Cargo.lock.

If those libraries didn't make breaking changes, then it'll be fine.

Reply:
and if they don't follow version semantics.